webapp/components/features/ReportsTable/ReportsTable.spec.js

```diff
@@ -34,7 +34,7 @@ describe('ReportsTable', () => {
 
     describe('given no reports', () => {
       beforeEach(() => {
-        propsData = { ...propsData, reports: [] }
+        propsData = { ...propsData }
```

Comment:
Because it increases coverage due to the fact that vue requires a function to set defaults. The default is set like this: `() => []` which is an uncovered line. With removing the value for reports, which reassembles the default value we increase coverage for the default function. I think this also effectively tests if the default value is still interpreted correctly and would allow us to change the default value without touching the test itself.

Reply:
Ah, now I understand.
Thanks for your explanation! 🙏🏼